package.json

```diff
@@ -4,7 +4,7 @@
   "description": "This is a Workshop about Cypress",
   "main": "index.js",
   "scripts": {
-    "test:open": "npm run lint && cypress open",
```

Comment:
you don't need to delete the `test:open` script

Reply:
@holgiosalos shall I restore the previous line with test:open? because in the first point i got asked to add this line so it was a bit confusing:

"scripts": {
    ... scripts existentes, no borrar
    **"test": "cypress run"**
    ... scripts existentes, no borrar
  },

thank you in advance

Reply:
yes the comments says that you should not delete the existing scripts

Reply:
just add the test: cypress run, but other existing scripts remain the same